### PR TITLE
chore: use endpointId if it exists in package.json

### PR DESCRIPTION
**Pull request by kaf-lamed-beyt**
@ericallam, sorry for the back and forth. I realized that there was a conflict in the last PR again. 

Here's a new one with an appropriate changeset

/claim #165

### CODE REVIEW

File: .changeset/dry-teachers-laugh.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
```
Comment (ericallam) [inline .changeset/dry-teachers-laugh.md]: Remove this file

File: .changeset/twelve-bobcats-collect.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@trigger.dev/cli": minor
```
Comment (ericallam) [inline .changeset/twelve-bobcats-collect.md]: This needs to be a "patch" change. Just change this file (need need to run `pnpm run changeset:add` again

**Review by ericallam — CHANGES_REQUESTED** [review]

**Comment by kaf-lamed-beyt** [comment]
Awesome! Will do these.

File: .changeset/twelve-bobcats-collect.md (hunk 2 above)
Comment (kaf-lamed-beyt) [reply .changeset/twelve-bobcats-collect.md]: @ericallam kindly take a look.

Comment (ericallam) [reply .changeset/twelve-bobcats-collect.md]: Okay now you just need to delete this file cause you've already got another changeset file

Comment (kaf-lamed-beyt) [reply .changeset/twelve-bobcats-collect.md]: Ah shoot! So after deleting it... I won't need to `pnpm changeset:add` again, yeah?

Comment (ericallam) [reply .changeset/twelve-bobcats-collect.md]: Nope, just delete this `.changeset/twelve-bobcats-collect.md` file and the PR will be good to go

Comment (kaf-lamed-beyt) [reply .changeset/twelve-bobcats-collect.md]: Awesome! I'll do that now.

Comment (kaf-lamed-beyt) [reply .changeset/twelve-bobcats-collect.md]: Pushed! 🚀

**Review by ericallam — APPROVED** [review]

**Pull request merged**